util/generate-sample-ledger/src/lib.rs

```diff
@@ -38,6 +38,8 @@ pub fn bootstrap_ledger(
     let mut db = LedgerDB::open(path.clone()).expect("Could not open ledger_db");
 
     let num_outputs: u64 = (recipients.len() * outputs_per_recipient_per_block * num_blocks) as u64;
+    assert!(num_outputs >= 16);
```

Comment:
Should this be `== 16` since `MAX_OUTPUTS` is 16?

Reply:
The origin block is the only case where we can create more outputs than 16 (that's how we make 100K outputs for testing).